modules/pubmaticBidAdapter.js

```diff
@@ -217,6 +219,108 @@ function _checkParamDataType(key, value, datatype) {
   }
 }
 
+function _createNativeRequest(params) {
+  var nativeRequestObject = {
+    assets: []
+  };
+  for (var key in params) {
+    var assetObj = {};
```

Comment:
need a check for params.hasOwnProperty(key)

Reply:
we are iterating keys on parmas only 
So they will exist

Reply:
when we loop on an object, loop control also loops to prototype properties, so it is a good practice to use hasOwnProperty to ensure that we loop on only direct properties and not prototype properties.

Reply:
Done put a check on key